blog/20140130-mate-desktop-live-cd.de.md

```diff
@@ -16,7 +16,7 @@ Diese Live-CD wurde erstellt, damit potentielle neue Nutzer den MATE-Desktop
 ausprobieren können, ohne Änderungen an Ihrem System vornehmen zu müssen.
 Das Image kann auf eine DVD gebrannt, als ISO eingehangen oder
 direkt auf einen USB-Stick (mittels eines Utilitys wie `dd`) geschrieben werden.
-[Linux Luddites](http://linuxluddites.com/) aufgepasst: `unetbootin` wird nicht unterstützt.
+[Linux Luddites](https://linuxluddites.com/) aufgepasst: `unetbootin` wird nicht unterstützt.
```

Comment:
https://linuxluddites.com/ seems to be dead page,

maybe https://luddites.latenightlinux.com/ is better? (episode 26 with Martin Wimpress interview)

Reply:
https://www.linuxluddites.com/ is working for me with firefox.

Reply:
yes, but it shows "hello world" page.

Reply:
See https://github.com/mate-desktop/mate-desktop.org/pull/164/files#r355193541